### PR TITLE
Add information about difference in behavior

### PR DESCRIPTION
**Pull request by tadeegan**
I would like to add more information about performance and behavior differences but I think this can be done incrementally.

### CODE REVIEW

**Comment by MatrixFrog** [comment]
lgtm. If Github will let you then I think you can probably go ahead and merge this.

File: README.md [hunk 1]
```diff
@@ -139,6 +139,9 @@ Unless you're using the Gulp or Webpack plugins, you'll need to specify code via
 Both `jsCode` and `externs` accept an array containing objects in the form `{src, path, sourceMap}`.
 Using `path`, you can construct a virtual filesystem for use with ES6 or CommonJS imports&mdash;although for CommonJS, be sure to set `processCommonJsModules: true`.
 
+## Transpilation
+The Javascript version of the Closure-Compiler is a transpiled by GWT from the Java source.  For more details on the differences in behavior see the [super sourced files](https://github.com/google/closure-compiler/tree/master/src/com/google/javascript/jscomp/gwt/super) in the main repo.
```
Comment (ericmdantas) [inline README.md]: *is a transpiled* > *is transpiled*

Comment (tadeegan) [inline README.md]: Thanks

**Comment by tadeegan** [comment]
I don't have merge access.

**Pull request merged**